licensing/locale/fr_FR/LC_MESSAGES/messages.po

```diff
@@ -832,7 +832,7 @@ msgstr "moins..."
 
 #: ajax_htmldata.php:223 ajax_htmldata.php:865 ajax_htmldata.php:983
 msgid "(none uploaded)"
-msgstr "(non téléchargé)"
+msgstr "(aucun téléchargé)"
```

Comment:
Fixed in https://github.com/Coral-erm/Coral/pull/85/commits/8978add0fd40231b96dc637754870e2b4f3370d3, used téléversé everywhere.